src/components/informationCard/InformationCard.css

```diff
@@ -0,0 +1,32 @@
+.infoCard {
+    flex: 0 1 calc(50% - 30px);
```

Comment:
We missen nog een media-query voor mobiel die ervoor zorgt dat de kaarten 100% breed worden :)

Reply:
Als volgt opgelost: 

```
@media screen and (max-width: 768px) {
    /*this screen width includes larger phones*/
    .info-card {
        flex: 0 1 calc(100% - 1.4rem);
    }

}
```